data/examples_loading.py

```diff
@@ -0,0 +1,19 @@
+import json
+from typing import List
+from pydantic import BaseModel #this should not be needed here, this will get cleaned up when its moved to its own examples.py
+import os
+from models.models import Example
+
+class Examples(BaseModel): #do we need BaseModel? What does it do? 
+    Examples: List[Example]

```

Comment:
We have Example class in models.py but Examples class here, for consistency let's move them both to models.py for now although we should probably sort that file out soon as it's getting bloated